docs/docsite/rst/dev_guide/developing_collections_testing.rst

```diff
@@ -19,7 +19,15 @@ Use ``ansible-test`` tool from your collection directory, which must include ``a
 
 You can run several sanity tests, as well as run unit and integration tests for plugins using ``ansible-test``. When you test collections, test against the ansible-core version(s) you are targeting.
 
-You must always execute ``ansible-test`` from the root directory of a collection. You can run ``ansible-test`` in Docker containers without installing any special requirements. The Ansible team uses this approach in Azure Pipelines both in the ansible/ansible GitHub repository and in the large community collections such as `community.general <https://github.com/ansible-collections/community.general/>`_ and `community.network <https://github.com/ansible-collections/community.network/>`_. The examples below demonstrate running tests in Docker containers.
+You must always execute ``ansible-test`` from the root directory of a collection. You can run ``ansible-test`` in Docker containers without installing any special requirements. The Ansible team uses this approach in Azure Pipelines both in the ansible/ansible GitHub repository and in the large community collections such as `community.general <https://github.com/ansible-collections/community.general/>`_ and `community.network <https://github.com/ansible-collections/community.network/>`_ to automatically run the tests when pull requests are submitted.
```

Comment:
While I don't think it will help with this review I would advocate for using one sentence per line here. Semantic linebreaks would make it so much easier to spot the differences here.

Reply:
I've only added `to automatically run the tests when pull requests are submitted.` to the end.
Using one sentence per line WFM though will require massive refactoring (and a dedicated PR).